services/opentsdb/service.go

```diff
@@ -200,6 +214,27 @@ func (s *Service) Close() error {
 	return nil
 }
 
+// VerifyReloadTLSCertificate verifies that the configured TLS certificate can be reloaded.
+// If so, then a function that will apply the reloaded certificate is returned. If no reload
+// action is necessary, then nil is returned for the reload function.
+func (s *Service) VerifyReloadTLSCertificate() (func() error, error) {
```

Comment:
Same thing as `CertLoader.VerifyReload`. Is this the best name for the method?

Reply:
`MakeTlsCertLoaderFunc`???  Just a suggestion.